go/ct/utils/adapter.go

```diff
@@ -35,6 +36,8 @@ func ToVmParameters(state *st.State) tosca.Parameters {
 		transactionContext = &st.TransactionContext{}
 	}
 
+	codeHash := tosca.Hash(sha3.Sum256(code))
```

Comment:
nit: the `state.Code` has a [Hash](https://github.com/Fantom-foundation/Tosca/blob/e20124038f600b26fcdfc91817ca2a2d31d0edc1/go/ct/st/code.go#L73) function that caches the result. Since the same code is used in some case thousands of times this would reduce the introduced runtime costs.

Also, technically the code hash is expected to be a `keccak256` hash, but I guess no interpreter is checking for that.